packages/core/src/sanitization/html_sanitizer.ts

```diff
@@ -271,7 +270,7 @@ export function _sanitizeHtml(defaultDoc: any, unsafeHtmlInput: string): Trusted
     const sanitizer = new SanitizingHtmlSerializer();
     const safeHtml = sanitizer.sanitizeChildren(
         getTemplateContent(inertBodyElement!) as Element || inertBodyElement);
-    if (isDevMode() && sanitizer.sanitizedSomething) {
+    if (ngDevMode && sanitizer.sanitizedSomething) {
```

Comment:
Since this code is shared between Ivy and ViewEngine, we should take into account the fact that `ngDevMode` may be undeclared, so this check should look like this:
```suggestion
    if ((typeof ngDevMode === 'undefined' || ngDevMode) && sanitizer.sanitizedSomething) {
```
See an [example here](https://github.com/angular/angular/blob/master/packages/forms/src/directives/ng_model.ts#L295) as well.

Reply:
Was thinking about that like "to do or not to do" :smile: Thanks for clarifying  why `typeof ngDevMode` is needed.

I have updated the code.

Reply:
Great, thanks for the update!